### ISSUE TITLE
Need a Verify Deployment method

### ISSUE THREAD

**Issue by etsauer** [redhat-cop/pipeline-library#15]
@redhat-cop/developer-workflow

**Comment by etsauer** [comment]
```
          openshift.withCluster( env.API, env.TOKEN ) {
            openshift.withProject("${STAGE3}") {
              def dcObj = openshift.selector('dc', env.APP_NAME).object()
              def podSelector = openshift.selector('pod', [deployment: "${APP_NAME}-${dcObj.status.latestVersion}"])
              podSelector.untilEach {
                  echo "pod: ${it.name()}"
                  return it.object().status.containerStatuses[0].ready
              }
            }
          }
```

**Comment by pabrahamsson** [comment]
Fixed by #16
